public/dashboards/home.json

```diff
@@ -3,26 +3,13 @@
   "panels": [
```

Comment:
Do you think it would be possible to duplicate this dashboard and load that instead (with these changes)?